curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/62017f47c87be96457c49f46.md

```diff
@@ -39,7 +39,7 @@ Your `span[class~="sr-only"]` selector should have the `-webkit-clip-path` prope
 
 ```js
 // this one gets removed apparently
-const webkitRegex = /;\s*-webkit-clip-path\s*:\s*inset\(\s*50%\s*\)\s*!important;/
+const webkitRegex = /[^a-zA-Z]-webkit-clip-path\s*:\s*inset\(\s*50%\s*\)\s*!important/
```

Comment:
This still won't pass if the property starts at the beginning of the line. And will pass many garbage prefixes.

Reply:
Actually it will pass if it's at the beginning of the line. It's checking for the absence of something. Not the inclusion.

Reply:
This passes:

```
@-webkit-clip-path: inset(50%) !important;
```

And this:

```
-webkit-clip-path: inset(50%) !importants;
```

Trying to catch all these user mistakes is not exactly easy using a regex. I do have something that I think does a pretty good job but the issue I just opened is going to make this PR obsolete, so I think I'll just keep it to myself :slightly_smiling_face:

Reply:
Aren't you requiring a single character prefix? I don't know any regex flavour where a character range would match the empty string.